stack/templates/_helpers.tpl

```diff
@@ -201,4 +201,9 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 
 {{- define "oidcProxy.authDomain" -}}
 {{ .Values.ingress.host }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+
+{{- define "blockCountries" -}}
+{{ default .Values.ingress.blockCountries  }}
```

Comment:
values.yaml supplies the default of `[]` so i don't think this template is necessary